x-pack/plugins/apm/common/alert_types.ts

```diff
@@ -74,39 +106,52 @@ export function formatTransactionErrorRateReason({
   measured,
   serviceName,
   asPercent,
+  windowSize,
+  windowUnit,
 }: {
   threshold: number;
   measured: number;
   serviceName: string;
   asPercent: AsPercent;
+  windowSize: number;
+  windowUnit: string;
 }) {
   return i18n.translate('xpack.apm.alertTypes.transactionErrorRate.reason', {
-    defaultMessage: `Failed transactions rate is greater than {threshold} (current value is {measured}) for {serviceName}`,
+    defaultMessage: `Failed transactions is {measured} in the last {interval} for {serviceName}. Alert when > {threshold}.`,
     values: {
       threshold: asPercent(threshold, 100),
       measured: asPercent(measured, 100),
       serviceName,
+      interval: getInterval(windowSize, windowUnit),
     },
   });
 }
 
 export function formatTransactionDurationAnomalyReason({
+  threshold,
   serviceName,
   severityLevel,
   measured,
+  windowSize,
+  windowUnit,
 }: {
+  threshold: number;
   serviceName: string;
   severityLevel: string;
   measured: number;
+  windowSize: number;
+  windowUnit: string;
 }) {
   return i18n.translate(
     'xpack.apm.alertTypes.transactionDurationAnomaly.reason',
     {
-      defaultMessage: `{severityLevel} anomaly detected for {serviceName} (score was {measured})`,
+      defaultMessage: `{severityLevel} anomaly with a score of {measured} was detected in the last {interval} for {serviceName}. Alert when {threshold}.`,
```

Comment:
@fkanout Do we possibly need Alert when `>` {thershold}?

Reply:
I will double-check that. But I believe we don't need the `>` sign as we are using **_word_** here

Reply:
I don't know what you mean with `word`, but `threshold` is a number, no?

Reply:
@dgieselaar in this use case the threshold is a word that represents the severity level. Starts from "info" to "critical"

Reply:
The type for this function says `number` though?  I'm a little confused.

Reply:
@dgieselaar, thank you for pointing that out. From a user perspective when creating a new rule the `threshold` is  what we call the `severityLevel`

Maybe we don't need to add the last part in the reason message _Alert when ..._ I need to ask @vinaychandrasekhar about that. As the `threshold` is